aws/components/apigateway/setup.ftl

```diff
@@ -252,15 +260,37 @@
             ] ]
     [/#if]
 
-    [#local accessLgId   = resources["accesslg"].Id]
-    [#local accessLgName = resources["accesslg"].Name]
+    [#if solution.AccessLogging.Enabled]
+        [#-- Manage Access Logs with Kinesis Firehose --]
```

Comment:
Need an extra attribute to control whether firehose or cloud watch log groups is used. Also need logic to permit existing installations to preserve the existing cloudwatch log groups.

So I think something like `"aws:Firehose: True"` and `"aws:KeepLogGroup" : true`. stageLogTarget is controlled via the former. The creation of the log group is performed if either Firehose=false or KeepLogGroup=true.

Then we can incrementally remove the log groups once the data has been processed via firehose as a separate activity.